README.md

```diff
@@ -19,14 +19,21 @@ Replace the placeholder with your Android SDK location, ex: `/User/{USER_NAME}/L
 
 [iOS Specific](https://github.com/realeyes-media/scte35-swift)
```

Comment:
I think this link to the iOS equivalent project can be removed. And the "Android Specific" and "Javascript Specific" links are to test harnesses as well so the "iOS Test Harness" can be moved under the same heading as these